modules/social_features/social_post/js/keycode-submit.js

```diff
@@ -10,29 +10,24 @@
   Drupal.behaviors.keycodeSubmit = {
     attach: function (context, settings) {
 
-      // Enable CTRL+Enter to submit a form.
-      var keySubmit = function (form, textarea, submit) {
-        textarea.on("keydown", function (e) {
-          // Make sure the textarea contains text.
-          if ($.trim(textarea.val()) != "") {
-            if ((e.keyCode == 13 && e.ctrlKey) || (e.keyCode == 13 && e.metaKey)){
-              e.preventDefault();
-              submit.prop('disabled', true);
-              form.submit();
-            }
-          }
-        });
-      }
-
       // Post form.
-      var postForm = $('#social-post-entity-form');
-      keySubmit($(postForm), $('.form-textarea', postForm), $('.form-submit', postForm));
-
-      // Comment forms.
-      $('.comment-form').each(function () {
-        keySubmit($(this), $('.form-textarea', this), $('.form-submit', this));
-      });
-
+        $('#social-post-entity-form .form-textarea')
+      .off('keydown', onTextAreaKeyDown)
+      .on('keydown', onTextAreaKeyDown);
+      // comment form
+        $('.comment-form .form-textarea').on('keydown')
+      .off('keydown', onTextAreaKeyDown)
+      .on('keydown', onTextAreaKeyDown);
```

Comment:
Could you explain why you used `off` here? If it is to ensure that we don't add the event handler multiple times, then we can also add the `context` variable as second parameter to our search (`$`) call to only search in the elements that were added when this function was triggered.

It also looks like the `.on('keydown')` for the comment form text area is a stray trigger.